src/indexing.c

```diff
@@ -30,8 +32,17 @@ get_tags(const char *fpath)
 
         file->path = fpath;
         file->tags->artist = taglib_tag_artist(tag);
+        if (strlen(file->tags->artist) == 0)
```

Comment:
how about we do something like `taglib_tag_artist(tag) ? taglib_tag_artist(tag) : untagged`?

Reply:
Would that actually work? It's not a null pointer, just an empty string.

Reply:
Right, we could use [0], but at this point, the code would look about the same